mono/mini/aot-compiler.c

```diff
@@ -1432,7 +1432,6 @@ arch_emit_got_access (MonoAotCompile *acfg, const char *got_symbol, guint8 *code
 #elif defined(TARGET_POWERPC)
 	{
 		guint8 buf [32];
-		guint8 *code;
 
 		emit_bytes (acfg, code, mono_arch_get_patch_offset (code));
 		code = buf;
```

Comment:
The change is good I think, that code is dead thats why nobody has ran into it before.